activejob/test/cases/test_helper_test.rb

```diff
@@ -15,6 +15,10 @@
 require "models/person"
 
 class EnqueuedJobsTest < ActiveJob::TestCase
+  setup do
+    skip unless adapter_is?(:test)
```

Comment:
On `main` the test adapter is always `:test` inside these tests (regardless of what adapter is set in ENV).

Reply:
Should add another set of test classes for non-test adapters.